src/TraceEvent/TraceLog.cs

```diff
@@ -5301,6 +5301,36 @@ public TraceProcess FirstProcessWithName(string processName, double afterTimeRel
             }
             return null;
         }
+
+        /// <summary>
+        /// Returns the process with the most amount of CpuMsec time.  Should this time be equal, the oldest ID is prioritized
+        /// </summary>
+        /// <param name="processName"> The process name to look for in the stacks </param>
+        /// <returns> The process with the greatest cpuMSec, or the oldest process if there was a tie. </returns>
+        public TraceProcess ProcessWithGreatestCpuMSec(string processName)
```

Comment:
The precedent in other user commands of this type is to use the last process in the trace with the specified name.  I can see that this might not be the right process.  That said, if you want to select based on CPU msec, I would recommend that you move this method to UserCommands.cs, so that it is private to the user command.  This isn't something that I think we should expose publicly in TraceEvent.

Reply:
That's fair.  My assumption here is if you want this data via the command line, you are likely investigating the process with the greatest cpu msec, should there be more than one of the same process.  My reasoning for this assumption is that the most costly process is likely the one you are investigating.  I'll add another parameter so you can choose to either do greatest msec or last process of that's fine (and also move this method)

Reply:
Sounds great.  Thanks @EmosewaMC!

Reply:
Resolved